packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -626,8 +633,9 @@ export function useMemo<T>(
       return prevState[0];
     }
   }
-
+  pauseHooks(true)
```

Comment:
Since we don't need `pauseHooks` elsewhere this can become local `isInUseMemo = true`.

Reply:
agreed, that was my first cut as well.